### ISSUE TITLE
Make it possible to run node with a docker exec in rh-che stack

### ISSUE THREAD

**Issue by l0rd** [redhat-developer/rh-che#371]
We currently need to prefix commands with scl (c.f. #189 )

**Comment by eivantsov** [comment]
@l0rd adding ENVs worked for me

```
ENV PATH=/opt/rh/rh-nodejs6/root/usr/bin${PATH:+:${PATH}}
ENV LD_LIBRARY_PATH=/opt/rh/rh-nodejs6/root/usr/lib64${LD_LIBRARY_PATH:+:${LD_LIBRARY_PATH}}
ENV PYTHONPATH=/opt/rh/rh-nodejs6/root/usr/lib/python2.7/site-packages${PYTHONPATH:+:${PYTHONPATH}}
ENV MANPATH=/opt/rh/rh-nodejs6/root/usr/share/man:$MANPATH
```

**Comment by l0rd** [comment]
Ok great thanks @eivantsov

**Comment by JPinkney** [comment]
Fixed as of eclipse/che-dockerfiles@8e70296cb5a9bd79add294b3ce1bd55daf414798
